coriolis/tasks/replica_tasks.py

```diff
@@ -284,15 +279,12 @@ def run(self, ctxt, instance, origin, destination, task_info,
             "migr_resources"]
 
         migr_connection_info = replica_resources_info["connection_info"]
-        migr_connection_info = base.marshal_migr_conn_info(
-            migr_connection_info)
-        schemas.validate_value(
-            migr_connection_info,
-            schemas.CORIOLIS_DISK_SYNC_RESOURCES_CONN_INFO_SCHEMA,
-            # NOTE: we avoid raising so that the cleanup task
-            # can [try] to deal with the temporary resources.
-            raise_on_error=False)
-
+        try:
+            backup_writers.BackupWritersFactory(
+                migr_connection_info, None).get_writer()
+        except BaseException as err:
+            raise exception.CoriolisException(
```

Comment:
WARN: Raising here will prevent the `task_info` from getting returned and thus the `DeleteReplicaTargetResourcesTask` from actually cleaning them up:
https://github.com/cloudbase/coriolis/blob/master/coriolis/tasks/replica_tasks.py#L309

Reply:
Indeed! We should not be raising here. We should let this pass, and fail inside the ```replicate_disks``` method, when the export provider attempts to instantiate the backup writer class using the connection info.

Changed it to ```LOG.exception```